### ISSUE TITLE
Issue with @Expose({ toPlainOnly: true }) Not Returning Data in Audit DTO

### ISSUE THREAD

**Issue by kauandotnet** [conceptadev/nestjs-modules#140]
I've encountered a problem with the AuditDto. The expected data isn't returned in API responses. Removing `toPlainOnly` resolves the issue.

Result:
```json
{
 "audit": {}
}
```

https://github.com/conceptadev/rockets/blob/1904d9117056cba7b70a57f949c3eba80d7ff5ed/packages/nestjs-common/src/audit/dto/audit.dto.ts#L10-L60

My solution:

```ts
@Exclude()
export class AuditDto extends _AuditDto {
  @Expose()
  dateCreated: Date;

  @Expose()
  dateUpdated: Date;
}
```

**Comment by MrMaz** [comment]
Need to replicate this on starter project

**Comment by MrMaz** [comment]
fixed by #142
